bazaDanych.trace.db

```diff
@@ -0,0 +1,71 @@
+2021-05-29 17:38:01 jdbc[14]: exception
```

Comment:
Czy ten plik jest potrzebny w tym PR?

Reply:
Wydaje mi się, że to jakiś "śmieciowy" plik, który niepotrzebnie zkomitowałem.